### DISCUSSION TITLE
What is the name of the cat in the film "Hocus Pocus"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#305] [Galaxy Trivia] [answered]
**Hint:** TB

**Comment by thuongtruong1009** [comment, marked as answer]
The name of the cat in the film "Hocus Pocus" is Thackery Binx. In the movie, Thackery was transformed into a black cat by the Sanderson sisters and was cursed to live forever as a feline until a virgin lit the Black Flame Candle on Halloween night. He befriends the protagonist, Max, and helps him and his friends defeat the Sanderson sisters.

**Comment by Mithun1508** [comment]
"Thackery Binx" the iconic and soon-to-be iconic supporting felines in Hocus Pocus and the new Disney+ sequel Hocus Pocus 2, respectively
